fl4health/clients/evaluate_client.py

```diff
@@ -88,7 +88,8 @@ def setup_client(self, config: Config) -> None:
         self.criterion = self.get_criterion(config)
         self.parameter_exchanger = self.get_parameter_exchanger(config)
 
-        self.metrics_reporter.add_to_metrics({"type": "client", "initialized": datetime.datetime.now()})
+        for r in self.reporters:
+            r.report({"host_type": "client", "initialized": str(datetime.datetime.now())})
```

Comment:
nit: Is it worth having some sort of ReporterManager that takes in reporting metrics and routes it to each one of the reporters so you can minimize the amount of for loops you have in the code (ie one for each time you have to report or initialize anything)

Reply:
I thought about this and kind of was 50/50 on it. My initial logic for not including one was that I thought it would be more lines of code than just manually doing it every time. Not sure if that assumption is still true. Happy to add one, or to add equivalent methods to basic client. What do you think?